boat_simulator/common/sensors.py

```diff
@@ -0,0 +1,130 @@
+from dataclasses import dataclass
+
+from typing import Optional
+from numpy.typing import NDArray
+
+
+from boat_simulator.common.types import Scalar, ScalarOrArray
+
+from boat_simulator.common.generators import (
+    ConstantGenerator,
+    MVGaussianGenerator,
+    GaussianGenerator,
+)
+
+WindSensorGenerators = Optional[MVGaussianGenerator | ConstantGenerator]
```

Comment:
I wanted to move this to types, but a circular import would have occurred :(

Reply:
I think Generators uses data types defined in `types.py`. I tried to include Generator classes in `types.py`. Circular import :(